python/tvm/relay/backend/graph_executor_codegen.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, mod, target):
         self._get_irmodule = self._mod["get_irmodule"]
         self._setup(mod, target)
 
-    def _setup(self, mod, target):
+    def _setup(self, mod, target: Dict[int, Target]):
```

Comment:
This type is stronger than the dynamic check below, perhaps you put it in while you were groking the code?

Reply:
Yeah I think this slipped through, I think this was maybe the ideal type